examples/plot_hf_hub.py

```diff
@@ -106,6 +109,22 @@
     create_remote=True,
 )
 
+# %%
+# Once uploaded, other users can download and use it, unless you make the repo
+# private. Given a repository's name, here's how one can download it:
+repo_copy = mkdtemp(prefix="skops")
+hf_hub.download(repo_id=repo_id, dst=repo_copy)
```

Comment:
The `skops` will have more tools than the ones interacting directly with the hub. Things to hep with persistence as well as a transformer helping with data type conversions maybe. That's why I'd like to keep the hub interaction module separate. I'm not set on the `hf_hub` name, anything signifying that it has to do with the HF hub is fine with me, but I don't really like the idea of exposing everything on the root level of a package, and I think namespaces make sense and they should be used (I know that's not what we do in other huggingface's libraries).

I do agree that `hf_hub` could be seen as what you suggest, i.e. alias for `huggingface_hub`. But I'm not sure what to replace it with.

Reply:
Understood, I agree. How about these:
- `from skops import hub` -> linked to the Hub, but without the `hf` which reminds me of `huggingface_hub` aliased. I don't know if you intend to have support for other hubs, but in that case support for these could also go in there.
- `from skops import hub_utils` -> same as above, also a viable name IMO
- `from skops import hf_hub_utils` -> has `hf` in it, but still understandable that it's a utility module from this package, rather than an alias to `huggingface_hub`.

WDYT? Again, not feeling strongly so feel free to go ahead with anything that you're comfortable with.